packages/renderer/src/plugins/router.ts

```diff
@@ -12,11 +22,65 @@ const routes: Array<RouteRecordRaw> = [
     name: 'Account',
     component: AccountPage,
   },
+  {
+    path: '/upload',
+    name: 'Upload',
+    component: UploadPage,
+  },
+  {
+    path: '/init',
+    name: 'Create Website',
+    component: InitPage,
+  },
+  {
+    path: '/admin',
+    name: 'Admin Website',
+    component: AdminPage,
+  },
+  {
+    path: '/music',
+    component: BuildingPage,
+  },
+  {
+    path: '/movies',
+    component: BuildingPage,
+  },
+  {
+    path: '/tv-shows',
+    component: BuildingPage,
+  },
+  {
+    path: '/about',
+    component: AboutPage,
+  },
+  {
+    path: '/privacy-policy',
+    component: PrivacyPolicyPage,
+  },
+  {
+    path: '/terms',
+    component: TermsPage,
+  },
+  {
+    path: '/release',
+    name: 'Release',
+    component: ReleasePage,
+    props: true,
+  },
 ];
 
 const routeur = createRouter({
   history: createWebHashHistory(),
   routes,
+  scrollBehavior() {
+    return {top: 0};
+  },
+});
+
+routeur.afterEach(to => {
+  const {stub} = to.query;
+  const {status} = useDevStatus();
+  status.value = import.meta.env.VITE_STATIC_MODE ? 'static' : (stub !== undefined ? 'static' : 'live');
 });
 
```

Comment:
It looks like this is breaking navigation. I had to comment out afterEach call to can navigate in static mode.  @julienmalard  Is the same thing happening to you?

Reply:
This was working for me...how exactly does it break for you?

Reply:
When I try to navigate in 'static' mode, it just doesn't work. The console shows this.

![image](https://github.com/user-attachments/assets/84a55bff-92d9-4de1-9fe4-8984f78df0c9)

Reply:
It used to, but I don't know exactly when or why it broke. I'll look into it again.

Reply:
Semms like that to.query.stub is always null, idk if the afterEach makes sense here, I suggest removing it and just handling it with the VITE_STATIC_VIEW env var. Also I think that the parentNode error from the above screenshot happens because if the current page uses the static.value, the rendering changes during navigation.

Reply:
Hello,
`stub` will be `true` if it is set as a query parameter in the url (`?stub`). This allows users to specify `stub` mode either through an environmental variable or through the url.
I'm not sure I understand the `parentNode` error; do you have a traceback?
Thanks!

Reply:
Hi! I don't know why the parentNode error doesn't appear anymore. And now I understand the stub parameter 😉, so no changes are necessary here. Thank you